welcome/legal-notice.adoc

```diff
@@ -6,24 +6,6 @@ include::modules/common-attributes.adoc[]
 [.lead]
 Copyright © 2020 Red Hat, Inc.
 
-The text of and illustrations in this document are licensed by Red Hat under a Creative Commons Attribution–Share Alike 3.0 Unported license ("CC-BY-SA"). An explanation of CC-BY-SA is available at http://creativecommons.org/licenses/by-sa/3.0/. In accordance with CC-BY-SA, if you distribute this document or an adaptation of it, you must provide the URL for the original version. Modified versions must remove all Red Hat trademarks.
+OpenShift documentation is licensed under Apache License 2.0 (https://www.apache.org/licenses/).
 
-Red Hat, as the licensor of this document, waives the right to enforce, and agrees not to assert, Section 4d of CC-BY-SA to the fullest extent permitted by applicable law.
-
-Portions adapted from https://github.com/kubernetes-incubator/service-catalog/ with modifications by Red Hat. Licensed under the link:https://www.apache.org/licenses/LICENSE-2.0[Apache License 2.0].
-
-Red Hat, Red Hat Enterprise Linux, the Red Hat logo, the Shadowman logo, JBoss, OpenShift, Fedora, the Infinity logo, and RHCE are trademarks of Red Hat, Inc., registered in the United States and other countries.
-
-Linux® is the registered trademark of Linus Torvalds in the United States and other countries.
-
-Java® is a registered trademark of Oracle and/or its affiliates.
-
-XFS® is a trademark of Silicon Graphics International Corp. or its subsidiaries in the United States and/or other countries.
-
-MySQL® is a registered trademark of MySQL AB in the United States, the European Union and other countries.
-
-Node.js® is an official trademark of Joyent. Red Hat Software Collections is not formally related to or endorsed by the official Joyent Node.js open source or commercial project.
-
-The OpenStack® Word Mark and OpenStack logo are either registered trademarks/service marks or trademarks/service marks of the OpenStack Foundation, in the United States and other countries and are used with the OpenStack Foundation's permission. We are not affiliated with, endorsed or sponsored by the OpenStack Foundation, or the OpenStack community.
-
-All other trademarks are the property of their respective owners.
```

Comment:
We should keep all of this trademark boilerplate (probably some of it is not needed but let's treat that as out of scope fo this issue).